plugins/dependencies/src/main/kotlin/org/gradle/kotlin/dsl/IsNotADependency.kt

```diff
@@ -0,0 +1,29 @@
+@file:Suppress("DeprecatedCallableAddReplaceWith")
+
+package org.gradle.kotlin.dsl
+
+import org.gradle.api.artifacts.Dependency
+import org.gradle.api.artifacts.dsl.DependencyHandler
+
+/**
+ * Marker interface that prevents the user to add a wrong dependency like
+ *
+ *     implementation(Testing.kotest)
+ *                   ^^^^^^^^^^^^^^^
+ *                   IDE: This is not a valid dependency notation but a group of dependency notations
+ */
+interface IsNotADependency
+
+const val ErrorIsNotADependency = """This is not a valid dependency notation but a group of dependency notations"""
```

Comment:
This should be private.
```suggestion
private const val ErrorIsNotADependency = """This is not a valid dependency notation but a group of dependency notations"""
```

Reply:
done, thanks